libs/platform/ebpf_epoch.c

```diff
@@ -216,10 +216,8 @@ ebpf_epoch_terminate()
     EBPF_LOG_ENTRY();
     uint32_t cpu_id;
 
-    ebpf_free_timer_work_item(_ebpf_flush_timer);
-    _ebpf_flush_timer = NULL;
-
     _ebpf_epoch_rundown = true;
+
```

Comment:
I don't think this fix is correct. Even if _ebpf_epoch_rundown is marked volatile, there is no guarantee that the DPC running the timer will have completed.